cfme/cloud/keypairs.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+import cfme.fixtures.pytest_selenium as sel
+from cfme.common import SummaryMixin, Taggable
+from cfme.web_ui import (Quadicon, flash, Form, Input, form_buttons, fill, summary_title,
+    breadcrumbs, AngularSelect)
+from functools import partial
+from cfme.web_ui import toolbar as tb
+from utils.wait import wait_for
+
+
+cfg_btn = partial(tb.select, "Configuration")
+keypairs_form = Form(
+    fields=[
+        ('name', Input("name")),
+        ('public_key', Input("public_key")),
+        AngularSelect('ems_id'),
```

Comment:
Okay, and this one is wrong, the test is failing because of that. It must always be a tuple, not just the item.